blackbox/docs/configuration.txt

```diff
@@ -1936,6 +1936,33 @@ to your config. This will prevent allocating primary and replica
 of the same shard on the same machine even if more than one instances
 running on it.
 
+Enterprise Features
+===================
+
+CrateDB Licensing
+-----------------
+
+The CrateDB licensing information.
+
+.. _license_enterprise:
+
+**license.enterprise**
+  | *Default:*  ``false``
+  | *Runtime:*  ``no``
+
+  The ``enterprise`` setting indicates whether the enterprise version of
+  CrateDB is used.
+
+.. NOTE::
+
+   To use any of the modules in this section, Crate.io must have given you
```

Comment:
i think refering to "this section" makes no sense here, i think we should refer to the setting